client/packages/common/src/intl/utils/DateUtils.ts

```diff
@@ -60,6 +60,18 @@ const formatIfValid = (
   }
 ): string => (isValid(date) ? format(date, dateFormat, options) : '');
 
+// Adds the current time to a date object (that presumably has 00:00 as its time
+// component)
+const addCurrentTime = (date: Date | null) => {
```

Comment:
```suggestion
const addCurrentTime = (date: Date | null): Date | null => {
```

Since this is a `lib` method, could you please use a doc comment instead (`/** */` )? and add a comment like: "Keeps the input date object unchanged."?

Reply:
None of the other methods in here have that format, what's the advantage of it? (Have added the additional comment though)

Reply:
You mean the docs comment? they show when you hover over the function in the IDE...

Reply:
![Screenshot 2023-04-18 at 11 22 35 AM](https://user-images.githubusercontent.com/5456533/232631472-6758947a-7bd6-4b8e-83f0-5d8463e93e25.png)

There's a big squiggly underline from the linter complaining about not specifying the @params and @return type. Which is unnecessary, as Typescript already gives that information. So we either:
- Add redundant info to please the linter 👎
- Change the linter rules 🤨 (something for another day)
- Leave as it was and how all the other functions are in here 👍

Reply:
Yes I am for removing the linter rule (@mark-prins ?) Workaround is to have an empty line between the comment and the method...

Reply:
Interesting, apparently the jsdoc rules in es-lint are now deprecated: https://eslint.org/blog/2018/11/jsdoc-end-of-life/

So I've turned them off for now. We can use [this plugin](https://github.com/gajus/eslint-plugin-jsdoc) if we want some jsdoc rules enforced.

6f46e3a6f